src/contour/Config.h

```diff
@@ -313,7 +313,7 @@ struct TerminalProfile
     ConfigEntry<bool, documentation::MouseHideWhileTyping> mouseHideWhileTyping { true };
     ConfigEntry<bool, documentation::SeachModeSwitch> searchModeSwitch { true };
     ConfigEntry<vtbackend::LineOffset, documentation::CopyLastMarkRangeOffset> copyLastMarkRangeOffset { 0 };
-    ConfigEntry<std::string, documentation::WMClass> wmClass { "contour" };
+    ConfigEntry<std::string, documentation::WMClass> wmClass {};
```

Comment:
You need to change it to `"org.contourterminal.Contour"` then, since we use this value to generate default config

Reply:
thx. fixed.